apps/src/code-studio/pd/application_dashboard/admin_edit_view.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Application Dashboard admin edit view
+ * Route: /:applicationId/edit
+ */
+
+import React, {PropTypes} from 'react';
+import ApplicationLoader from './application_loader';
+import FormDataEdit from './form_data_edit';
+
+export default class AdminEditView extends React.Component {
+  static propTypes = {
+    params: PropTypes.shape({
+      applicationId: PropTypes.string.isRequired
+    }).isRequired
+  };
+
+  renderApplication = ({applicationData, handleUpdate}) => (
+    <FormDataEdit
+      applicationId={this.props.params.applicationId}
+      applicationData={applicationData}
+      onUpdate={handleUpdate}
+    />
+  );
+
+  render() {
+    return (
+      <ApplicationLoader
+        applicationId={this.props.params.applicationId}
+        renderApplication={this.renderApplication}
+        loadRawFormData={true}
```

Comment:
Yes, in the normal [detail view](https://github.com/code-dot-org/code-dot-org/pull/20520/files#diff-b7fe47c0f7ee4d7192e9a13c3819e605R50). It's not explicitly set to false there, I just left that as a default and added this as the exception. In most cases we're more interested in the materialized "full answers" with some questions filtered out and additional text fields merge in, rather than the raw form data. See the [serializer changes](https://github.com/code-dot-org/code-dot-org/pull/20520/files#diff-0223b0b0b94bec827c3e95ddc1e438d1R20). Just for this form data edit, we need to view it in its raw form, which is an exception.